clear-site-data/storage.https.html

```diff
@@ -45,6 +70,33 @@
                 }, test_name);
               });
 
+              promise_test(function() {
+                const serviceWorkerResponseBody = TestUtils.getServiceWorkerControlledResponseBody();
+
+                // Tell the iframe to make a request for the URL that the service worker
+                // responds to
+                serviceWorkerTestPageIFrame.contentWindow.postMessage(null, "*");
+
+                return serviceWorkerResponseBody.then(function(body) {
+                  assert_equals(body, "FROM_NETWORK", "Response should not be 200");
+                });
+              }, "Service worker no longer responds to requests");
+
+              promise_test(function() {
+                const halfSecondTimeout = new Promise(function(resolve, reject) {
+                  step_timeout(() => {
+                    reject("controllerchange event was not fired");
+                  }, 500);
```

Comment:
I think the general guidance is to just rely on the overall test timeout mechanism instead of adding these kind of arbitrary timeouts in the test itself.

Reply:
I see, I've removed my arbitrary timeout